tests/xharness/Xharness.Tests/Tests/CrashReportSnapshotTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using Xharness.Execution;
+using Xharness.Execution.Mlaunch;
+using Xharness.Logging;
+
+namespace Xharness.Tests {
+	[TestFixture]
+	public class CrashReportSnapshotTests {
+		readonly string mlaunchPath = "./mlaunch";
+		string tempXcodeRoot;
+		string symbolicatePath;
+		
+		Mock<IProcessManager> processManager;
+		Mock<ILog> log;
+		Mock<ILogs> logs;
+
+		[SetUp]
+		public void SetUp ()
+		{
+			processManager = new Mock<IProcessManager> ();
+			log = new Mock<ILog> ();
+			logs = new Mock<ILogs> ();
+
+			tempXcodeRoot = Path.Combine (Path.GetTempPath (), Guid.NewGuid ().ToString());
+			symbolicatePath = Path.Combine (tempXcodeRoot, "Contents", "SharedFrameworks", "DTDeviceKitBase.framework", "Versions", "A", "Resources");
+
+			// Create fake place for device logs
+			Directory.CreateDirectory (tempXcodeRoot);
+
+			// Create fake symbolicate binary
+			Directory.CreateDirectory (symbolicatePath);
+			File.Create (Path.Combine (symbolicatePath, "symbolicatecrash"));
+		}
+
+		[TearDown]
+		public void TearDown () {
+			Directory.Delete (tempXcodeRoot, true);
```

Comment:
Should we also delete 'symbolicatePath'

Reply:
It is under the tempXcodeRoot so will be deleted with it